content/blog/monthly-january-2020.md

```diff
@@ -0,0 +1,68 @@
+
+You can review the changelogs here:
+
+*   [3.0.9](https://github.com/tikv/tikv/releases/tag/v3.0.9)
+*   [4.0.0-beta](https://github.com/tikv/tikv/releases/tag/v4.0.0-beta)
+
+## Reading materials
+
+[@brson](https://github.com/brson) published his first episode of the Rust Compile time series, [The Rust Compilation Calamity](https://tikv.org/blog/rust-compilation-calamity/), and shared his research and experiences with Rust compile times, using the TiKV project as a case study.
+
+[@BusyJay](https://github.com/BusyJay) (the first author) together with [@zhexuany](https://github.com/zhexuany) and [@bb7133](https://github.com/bb7133) (second authors) wrote [an overview on Raftstore in the series of A TiKV Source Code Walkthrough (in Chinese)](https://pingcap.com/blog-cn/tikv-source-code-reading-17/). This article uses the code of TiKV 3.0 version as an example to present the key definitions and designs in the raftstore source code.
+
+## Notable PRs
+
+- [@yiwu-arbug](https://github.com/yiwu-arbug) added S3 support to external_storage crate in [#6209](https://github.com/tikv/tikv/pull/6209).
+
+- [@breeswish](https://github.com/breeswish) integrated dashboard into PD in [#2086](https://github.com/pingcap/pd/pull/2086).
+
+- [@sticnarf](https://github.com/sticnarf) replaced all the readpools with [yatp](https://github.com/tikv/yatp) in [#6375](https://github.com/tikv/tikv/pull/6375) and [#6401](https://github.com/tikv/tikv/pull/6401).
+
+- [@MyonKeminta](https://github.com/MyonKeminta) implemented green GC in [#6138](https://github.com/tikv/tikv/pull/6138), [#6070](https://github.com/tikv/tikv/pull/6070) and [#6333](https://github.com/tikv/tikv/pull/6333).
+
+## Notable issues
+
+**Help wanted issues** (mentoring available)
+
+- [@brson](https://github.com/brson) opened the issue [#6402](https://github.com/tikv/tikv/issues/6402) regarding organizing and tracking progress on abstracting TiKV over generic storage engines.
+
+- [@yiwu-arbug](https://github.com/yiwu-arbug) created the issue [#6496](https://github.com/tikv/tikv/issues/6496), requesting a feature to make RocksDB issues easy to investigate.
+
+- [@yiwu-arbug](https://github.com/yiwu-arbug) suggested enabling `rate-bytes-per-sec` in [#6484](https://github.com/tikv/tikv/issues/6484) because it would keep RocksDB background compaction steady, smoothing both IO and CPU usage.
+
+**Call for participation**
+
+- [@yeya24](https://github.com/yeya24) suggested in [#6407](https://github.com/tikv/tikv/issues/6407) that some counter metrics don’t follow  the[ Prometheus Label Naming](https://prometheus.io/docs/practices/naming/) conventions and we should consider renaming them. While we agreed it is worth regulating the metrics, there would be some concerns along the way. What do you think?
```

Comment:
Could you explain a little bit what kind of concerns we have?

Reply:
Just there are too many of them in multiple places... updated.